lib/nexpose/password_policy.rb

```diff
@@ -0,0 +1,58 @@
+module Nexpose
+  require 'json'
+  # Configuration structure for password policies.
+  class PasswordPolicy < APIObject
+
+    attr_accessor :policyName
+    attr_accessor :minLength
+    attr_accessor :maxLength
+    attr_accessor :capitals
+    attr_accessor :digits
+    attr_accessor :specialChars
+
+    def initialize(policyName:, minLength:, maxLength:, specialChars:, capitals:, digits:)
```

Comment:
let's refactor all the attrs, and method arguments to use snake_case instead of camelCase